books/computer-science/bake

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env ruby
+
+# frozen_string_literal: true
+
+require 'bundler/inline'
+
+gemfile do
+  gem 'openstax_kitchen', ENV['USE_LOCAL_KITCHEN'] ? { path: '/code/kitchen' } : '11.2.0'
+  gem 'slop', '4.8.2'
+  gem 'byebug'
+end
+
+recipe = Kitchen::BookRecipe.new(book_short_name: :computer_science) do |doc|
+  include Kitchen::Directions
+
+  # Set overrides
+  doc.selectors.override(
+    page_summary: 'section.section-summary'
+  )
+
+  book = doc.book
+  metadata = book.metadata
+
+  # Some stuff just goes away
+  book.search('cnx-pi').trash
+
+  BakePreface.v1(book: book, title_element: 'h1')
+  BakeChapterIntroductions.v1(book: book)
+  BakeChapterTitle.v1(book: book)
+  BakeUnitTitle.v1(book: book)
+  BakeUnitPageTitle.v1(book: book)
+
+  BakeAutotitledNotes.v1(
+    book: book,
+    classes: %w[concepts-practice tech-everyday global-tech link-to-learning industry-spotlight]
+  )
+
+  BakeAutotitledNotes.v1(
+    book: book,
+    classes: %w[think-through],
+    bake_exercises: true
+  )
+
+  BakeUnclassifiedNotes.v1(book: book)
+
+  book.chapters.each do |chapter|
+    BakeLearningObjectives.v2(chapter: chapter, add_title: false)
+    BakeNonIntroductionPages.v1(chapter: chapter)
+    BakeChapterGlossary.v1(chapter: chapter, metadata_source: metadata)
+    BakeChapterSummary.v1(chapter: chapter, metadata_source: metadata, klass: 'section-summary')
```

Comment:
I think we are using `MoveCustomSectionToEocContainer` for section summary now :thinking: That's what I see in the newest recipes for other books. For example in Anthropology.

Reply:
🤔 should `BakeChapterSummary` be deprecated?

Reply:
Good question :thinking: I would say yes... But I'm not sure. I think Kendra can say more bout it. She created `MoveCustomSectionToEocContainer` and I think the goal was to deprecate all modules for specific sections.

Reply:
Awesome, thanks for the clarification K!